frontend/pages/admin/IntegrationsPage/cards/Mdm/Mdm.tsx

```diff
@@ -117,7 +117,7 @@ const Mdm = (): JSX.Element => {
       return <DataError />;
     }
 
-    if (!mdmApple || showCSRFlag) {
```

Comment:
This is used to make the CSR request flow act as it will when the real API is hooked up. Do you have a need for removing this now? If not, please leave it, and I'll remove it (and the other places it is used) when I'm wiring up the real API call.

Reply:
Slack thread resolution: Not touching this since it's not part of my feature